lib/stringify.js

```diff
@@ -1,35 +1,49 @@
 'use strict';
 
 const assert = require('assert');
+const JSON5 = require('json5');
 
 const format = require('./format');
 
+const DEFAULTS = { exportType: 'json5', space: 4 };
+
+const generators = {
+    json5: (obj, space) => JSON5.stringify(obj, null, space),
+    json: (obj, space) => JSON.stringify(obj, null, space),
+    commonjs: (obj, space) => `module.exports = ${JSON.stringify(obj, null, space)};\n`,
+    es2015: (obj, space) => `export default ${JSON.stringify(obj, null, space)};\n`
+};
+// Aliases
+generators.es6 = generators.es2015;
+
 /**
  * Create string representation of declaration
  *
- * @param  {Array|Object} decl source declaration
- * @param  {Object} opts  additional options
- * @param  {String} opts.format format
- * @return {String}       string representation
+ * @param {BemCell[]} decl - source declaration
+ * @param {Object} opts - additional options
+ * @param {String} opts.format - format of declaration (v1, v2, enb)
+ * @param {String} [opts.exportType=json5] - defines how to wrap result (commonjs, json5, json, es6|es2015)
+ * @param {String|Number} [opts.space] - number of space characters or string to use as a white space
+ * @returns {String}
  */
 module.exports = function (decl, opts) {
-    opts || (opts = {});
-    assert(opts.format, 'You must declare target format');
+    const options = Object.assign({}, DEFAULTS, opts);
 
-    Array.isArray(decl) || (decl = [decl]);
+    assert(options.format, 'You must declare target format');
+    assert(generators.hasOwnProperty(options.exportType), 'Specified format isn\'t supported');
 
-    const formatedDecl = format(decl, { format: opts.format });
-    const jsonStr = JSON.stringify(formatedDecl, null, 4);
-
-    let exportsStr = '';
-
-    if (opts.format === 'v1') {
-        exportsStr = 'exports.blocks';
-    } else if (opts.format === 'enb') {
-        exportsStr = 'exports.deps';
-    } else {
-        exportsStr = 'module.exports';
-    }
+    Array.isArray(decl) || (decl = [decl]);
 
-    return `${exportsStr} = ${jsonStr};\n`;
+    const fieldByFormat = {
+        v1: 'blocks',
+        enb: 'decl',
+        v2: 'deps'
+    };
```

Comment:
put this outside. Maybe even standalone JSON file. And write a good doc about - `What the f*ck is this`